restapi/src/main/java/io/stargate/web/docsapi/service/QueryInfo.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.web.docsapi.service;
```

Comment:
is this the correct package for this?

Reply:
you're right, perhaps I ought to move it to `io.stargate.web.docsapi.models`, would that fix better?

Reply:
moved